transformations/multilingual_dictionary_based_code_switch/transformation.py

```diff
@@ -92,20 +88,26 @@ def __init__(self, seed=0, max_outputs=1, code_switch_rate=0.9):
         self.switch_dict = load_dict()
 
     def generate(self, sentence: str):
-
-        return [code_switch(sentence, self.switch_dict, self.code_switch_rate, self.seed)]
+        words = sentence.split(" ")
+        out = ""
+        for word in words:
+            out += code_switch(word, self.switch_dict, self.code_switch_rate, self.seed)
+        return [out]

```

Comment:
You might want to support the `max_outputs` attribute by using another loop. I believe the `random.seed(seed)` call should happen at the top of this function then.

Reply:
@samsontmr Yes, samson. We have fixed it.